composer.json

```diff
@@ -8,7 +8,7 @@
         "jetbrains/phpstorm-stubs": "2019.2",
         "nikic/php-parser": "^4.2.1",
         "phpdocumentor/reflection-docblock": "^4.3.2",
-        "phpdocumentor/type-resolver": "^0.4.0",
+        "phpdocumentor/type-resolver": "^0.4.0 || ^1.0.0",
```

Comment:
Please just bump it: let's not keep the old one around.

Reply:
Ok. This is done in my last commit: dd1173567d2d4b850c9ff529d88170aaaf7480b6.
Thanks for the quick answer. I let you mark the code review as resolved if you think it is :)